unlock-app/src/pages/provider.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types'
```

Comment:
Previously `Web3Provider.js`.

Reply:
hum, not sure I follow? This is the `/provider` page, right?

Reply:
Yes, used to be:
```
import Provider from './Web3Provider'
<Route path="/provider" component={Provider} />
```